### ISSUE TITLE
Nightly updater

### ISSUE THREAD

**Issue by Tajnymag** [cSploit/android#308]
It would be cool to have cSplit automatically check for changes in nightly builds and serve the updated apk as soon as newer version is available. It's quite frustrating to download new apk every time I want to use cSploit (you are very hard working and every day there's a bunch of new fixes and features :) )

**Comment by daniloscala** [comment]
yes , nice if csploit nightly have number or show ins sites changes or updates , or automatic.

**Comment by hipopcorn** [comment]
In the about tab de van see the date of the nightly but it would be better  we can see it in the site

**Comment by tux-mind** [comment]
ok, requires #315

**Comment by michaeljhopkins** [comment]
Adding my +1 tot his
